docs/source/querying/querybuilder/intro.rst

```diff
@@ -16,7 +16,7 @@ Before starting to write a query, it helps to:
     Nodes of an AiiDA graph (vertices) are connected with links (edges).
     A *Node* can be either input or output of another *Node*, but also an
     ancestor or a descendant.
```

Comment:
Do we still use 'ancestor' and 'descendant'? Aren't we using 'successor' and 'predecessor' now, or something less family-like? Just a question, since I cannot remember - I can only remember there was a discussion about this.

Reply:
Valid keywords for queries are: 'with_incoming', 'with_outgoing', 'with_descendants', 'with_ancestors'